install.ps1

```diff
@@ -219,117 +212,129 @@ function Install-Mondoo {
   # determine download url
   $filetype = $DownloadType
   # cnquery and cnspec only ship as zip
-  if($Product -ne 'mondoo') {
+  If ($Product -ne 'mondoo') {
     $filetype = 'zip'
   }
+  # get the installed mondoo version
+  $installed_version = Get-WmiObject -Class Win32_Product | where vendor -eq "Mondoo, Inc."
 
+  $arch = 'amd64'
   $releaseurl = ''
-  If(![string]::IsNullOrEmpty($Version)) {
-    # specific version
-    $releaseurl = determine_version $Product $filetype $Version
-  } else {
+  $version = $Version
+
+  If ([string]::IsNullOrEmpty($version)) {
     # latest release
-    $releaseurl = determine_latest $Product $filetype
+    $version = determine_latest $Product $filetype $arch
```

Comment:
we determine the version but do not use it in the next line for $releaseurl, is that intentional?